python/shipDigiReco.py

```
@@ -75,43 +83,93 @@ def __init__(self, finput, fout, fgeo) -> None:
-        self.fitter = ROOT.genfit.DAF()
-        self.fitter.setMaxIterations(50)
-        if global_variables.debug:
-            self.fitter.setDebugLvl(1)  # produces lot of printout
-        # set to True if "real" pattern recognition is required also
 
         # for 'real' PatRec
         shipPatRec.initialize(fgeo)
 
+        ##Set up ACTS tracking geometry and magnetic field
+        self.detector = acts.examples.StrawtubeBuilder()
+        self.trackingGeometry = detector.trackingGeometry()
```

Ruff (0.15.12)

[error] Undefined name `detector`

(F821)